src/vs/editor/contrib/suggest/browser/suggestWidget.ts

```diff
@@ -477,9 +477,7 @@ export class SuggestWidget implements IContentWidget, IDisposable {
 	}
 
 	private onModelModeChanged(): void {
-		const model = this.editor.getModel();
-		const supports = SuggestRegistry.all(model);
-		this.shouldShowEmptySuggestionList = supports.some(s => true);
+		this.shouldShowEmptySuggestionList = true;
```

Comment:
all of this should be nuked. since the value of `shouldShowEmptySuggestionList` is now always true it should disappear, also we don't need to listen of mode change events anymore